src/objects/control.ts

```diff
@@ -86,8 +153,23 @@ export default class Control {
     return new Control(unflatten(flattened));
   }
 
-  // WIP - provides the ability to get the control in its raw form
-  toString() {
+  /**
+   * Converts the control object to a string representation in a specific format.
+   * WIP - provides the ability to get the control in its raw form
```

Comment:
I wonder if this is still WIP..?
```suggestion
   * Provides the ability to get the control in its raw form
```